Source/QuantityTypes/Units.csv

```diff
@@ -123,6 +123,7 @@ Frequency,RevolutionsPerMinute,rpm,1.0 / 60
 //Angular Frequency
 AngularFrequency,RadiansPerSecond,rad/s,1
 AngularFrequency,DegreesPerSecond,deg/s,Math.PI / 180
+AngularFrequency,DegreesPerSecondSymbol,°/s,Math.PI / 180
```

Comment:
it seems also to be possible to specify additional units using "|", as on line https://github.com/QuantityTypes/QuantityTypes/blob/main/Source/QuantityTypes/Units.csv#L94